triton-client/tsconfig.json

```diff
@@ -13,9 +24,9 @@
         "resolveJsonModule": true,
         "isolatedModules": true,
         "noEmit": true,
+        "noImplicitAny": false,
```

Comment:
That should ideally be true. But if the number of errors is too many in older codes, then maybe we can keep it.

Reply:
But even if the were tooo many errors, then it's based to put exceptions at the file or function level.

Reply:
this was one of the things that slipped through my refactoring, i put it back to its initial value